modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ObjectDetails.scss

```diff
@@ -21,8 +21,12 @@
 	position: relative;
 }
 
-.lfr__object-web-edit-object-details-external-data-source-container-beta {
-	left: 200px;
-	margin-top: 7px;
-	position: absolute;
+.lfr__object-web-edit-object-details-external-data-source-panel {
+	align-items: center;
+	display: flex;
+	flex-direction: row;
```

Comment:
Hi @Artartur, as the default value of flex-direction is row, could you check if this row is really necessary?

Reply:
Yeah, you're right. I've changed, thx Igor.